pages/Custom theme.md

```diff
@@ -4,8 +4,22 @@ platform:: [[All Platforms]] but some ways are not supported for [[Publish Web]]
 - ## Usage
 	- There are three ways to install custom themes
 		- 1. Load a theme from the [[Marketplace]].
+			- Click on top-right dot menu dropdown
+			- Open Plugins tab
+			- Click on [[Marketplace]] and the [[theme]]s tab 
```

Comment:
I'm going to remove these links as they have different meanings in this context. The linked words here refer to ui elements and not higher-level concepts. These words could change